aws-analytics-pinpoint/src/androidTest/java/com/amplifyframework/analytics/pinpoint/PinpointAnalyticsInstrumentationTest.kt

```diff
@@ -53,11 +54,18 @@ class PinpointAnalyticsInstrumentationTest {
         val context = ApplicationProvider.getApplicationContext<Context>()
         @RawRes val resourceId = Resources.getRawResourceId(context, CREDENTIALS_RESOURCE_NAME)
         val userAndPasswordPair = readCredentialsFromResource(context, resourceId)
-        synchronousAuth.signOut()
-        synchronousAuth.signIn(
-            userAndPasswordPair!!.first,
-            userAndPasswordPair.second
-        )
+        try {
```

Comment:
We should not be leaking state between tests.

Reply:
reverted as it passes locally without this change